mintapi/api.py

```diff
@@ -846,6 +851,18 @@ def get_invests_json(self):
         else:
             logger.error("FAIL2")
 
+    def get_investment_data_new(self):
+        investments = self.call_investments_endpoint()["Investment"]
+        for i in investments:
+            i.pop("metaData", None)
+        return investments
+
+    def call_investments_endpoint(self):
```

Comment:
Can we make this a private function? I think a goal for mintapi would be to be clear about what the public api is so we know what is a breaking change or not.